.pa11yci.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Could you update this file based on https://github.com/pa11y/pa11y-ci/issues/48#issuecomment-371184682? It'll need a new `.pa11yci.conf.js` file created, but it'll allow for the URL(s) to be scanned to be dynamic.

Whatever environment variable you decide to add will need added to the `README.md` file and added to GitHub Actions.

This'll also require a small change to the `pa11y-ci` command in `package.json`. It'll need to be `pa11y-ci -c .pa11yci.conf.js` to make use of the new config file